ramls/request-move.json

```diff
@@ -3,9 +3,15 @@
   "type": "object",
   "description": "Request to move a request from one item to another",
   "properties": {
+    "id": {
+      "description": "Id of request that have to be moved",
+      "type": "string",
+      "$ref": "uuid.json"
+    },
```

Comment:
I believe the `id` of the request comes from a template parameter in the path, not from the body of the request (see my general comment).

Reply:
Aha, thanks, have not seen the parameter. Will remove it.